pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -797,8 +797,13 @@ func (ctrl *PersistentVolumeController) updateClaimStatus(ctx context.Context, c
 			claimClone.Status.Capacity = nil
 			dirty = true
 		}
+		if utilfeature.DefaultFeatureGate.Enabled(features.VolumeAttributesClass) &&
+			claim.Status.CurrentVolumeAttributesClassName != nil {
+			claimClone.Status.CurrentVolumeAttributesClassName = nil
+			dirty = true
```

Comment:
Will claimClone.Status ever be nil?

Reply:
`status` is not a pointer type. https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L468